src/ChipCarousel/README.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 'ChipCarousel'
+type: 'component'
+components:
+- ChipCarousel
+categories:
+- Content
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+notes: |
+---
+
+``ChipCarousel`` component creates a scrollable horizontal block of chips with buttons for navigating to the previous and next set of chips.
+
+## Basic Usage
+
+```jsx live
+  <ChipCarousel
+    ariaLabel="example chip carousel"
+    items={Array.from({ length: 40 },
+      (_, index) => (
+        <Chip
+          key={index}
+          onClick={() => console.log(`Chip #${index + 1} clicked`)}
+        >
+          Chip #{index + 1}
+        </Chip>
+      )
+    )}
+  />
+```
+
+## `Chip` Carousel
+
+```jsx live
+<OverflowScroll ariaLabel="example chip carousel" hasInteractiveChildren>
+  <OverflowScrollContext.Consumer>
+    {({
+      setOverflowRef,
+      isScrolledToStart,
+      isScrolledToEnd,
+      scrollToPrevious,
+      scrollToNext,
+    }) => (
+      <>
+        <div className="mb-3">
+          <Button
+            onClick={scrollToPrevious}
+            className="mr-2"
+            size="sm"
+            disabled={isScrolledToStart}
+          >
+            Previous
+          </Button>
+          <Button
+            onClick={scrollToNext}
+            size="sm"
+            disabled={isScrolledToEnd}
+          >
+            Next
+          </Button>
+        </div>
+        <div ref={setOverflowRef} className="d-flex">
+          <OverflowScroll.Items>
+            <Chip iconAfter={Close}>New</Chip>
```

Comment:
I propose to simulate an array in this way. This will help us keep the example compact.

```
{[...Array(12).keys()].map(item => (
  <Chip key={item + 1} iconAfter={Close}>New</Chip>
))}
```